consistency-checker/src/test/java/com/redhat/lightblue/migrator/consistency/ITCaseConsistencyCheckerTest.java

```diff
@@ -0,0 +1,70 @@
+package com.redhat.lightblue.migrator.consistency;
+
+import static com.redhat.lightblue.util.test.AbstractJsonNodeTest.loadJsonNode;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertNull;
+
+import java.util.List;
+
+import org.junit.Test;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+
+public class ITCaseConsistencyCheckerTest extends AbstractMigratorController {
+
+    private final ConsistencyChecker consistencyChecker;
+    private static boolean initialized = false;
+
+    private static String versionMigrationJob;
+    private static String versionMigrationConfiguration;
+
+    public ITCaseConsistencyCheckerTest() throws Exception {
+        super();
+
+        if (!initialized) {
+            loadData("migrationConfiguration", versionMigrationConfiguration, "./test/data/load-migration-configurations.json");
```

Comment:
You can setup junit to fork, which I think does something with multithreading.  Maybe not per class though.